src/main/java/com/yy5/employee/service/EmployeeService.java

```diff
@@ -25,8 +26,8 @@ public Employee findEmployee(int employeeNumber){
                 .orElseThrow(() -> new EmployeeNotFoundException("EmployeeNumber:" + employeeNumber +" is not found"));
     }
 
-    public Employee insert (String name, int age ) {
-        Employee employee = new Employee(name ,age);
+    public Employee insert (String name, Object age ) throws MethodArgumentNotValidException {
```

Comment:
Object 型使うのはよっぽどの理由があるときにしたいです。
```suggestion
    public Employee insert (String name, int age ) {
```
これだと a みたいな値でもとりえるようになってしまいます。
あとここで MethodArgumentNotValidException が発生する可能性はないかと。

Reply:
ageは整数以外受け付けないので、int型に修正しました。
例外処理のthorwも削除しました。[62cdcb0](https://github.com/YYprogramer/employee/pull/12/commits/62cdcb083238bd1310bbea07f52e0eabb7b1a144)